tests/README.md

```diff
@@ -146,7 +146,111 @@ with the `zfs-tests.sh` wrapper script will look something like this:
     Results Summary
     SKIP	  52
     PASS	 1129
-    
+
     Running Time:	02:35:33
     Percent passed:	95.6%
     Log directory:	/var/tmp/test_results/20180515T054509
+
+### 4) Example of adding and running test-case (zpool_example)
+
+  This broadly boils down to 5 steps
+  1. Create/Set password-less sudo for user running test case.
+  2. Edit configure.ac, Makefile.ac appropriately
+  3. Create/Modify .run files
+  4. Create actual test-scripts
+  5. Run Test case
+
+  Will look at each of them in depth.
+
+  * Set password-less sudo for 'Test' user as test script cannot be run as root
+  * Edit file **configure.am** and include line under AC_CONFIG_FILES section
```

Comment:
```suggestion
  * Edit file **configure.ac** and include line under AC_CONFIG_FILES section
```

Reply:
Thanks for review! Resolved in new push.